### PR TITLE
Hijack with a contributor status

### PR DESCRIPTION
**Pull request by hhsnopek**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,11 @@
   "name": "static-lite",
   "repository": "https://github.com/BrandonRomano/static-lite.git",
   "author": "Brandon Romano",
+  "contributors": {
+    "name": "Henry Snopek",
+    "email": "hhsnopek@gmail.com",
+    "url": "hhsnopek.com"
+  },
```
Comment (BrandonRomano) [inline package.json:9]: Ahh you know what, this is totally going to bork all of my README line number references 😢 

Did you want to also update them?

**Review by BrandonRomano — CHANGES_REQUESTED** [review]

**Comment by BrandonRomano** [comment]
Also to give anyone else in the world context on this PR -- While @hhsnopek doesn't have any commits in the repo right now he's been involved largely in conversations regarding the architecture of it.

File: package.json (hunk 1 above)
Comment (hhsnopek) [reply package.json:9]: totally just moving my lines to the bottom haha

Comment (BrandonRomano) [reply package.json:9]: Lol naw keep these at the top 😜

Comment (hhsnopek) [reply package.json:9]: haha I forgot to push the changes up last night - I'll get these up tonight!

**Review by BrandonRomano — APPROVED** [review]
Thank you!

**Pull request merged**